src/carousel/carousel.scss

```diff
@@ -12,17 +12,29 @@
 
 @mixin iui-carousel-slider {
   display: flex;
+  gap: $iui-s;
   list-style: none;
   margin: 0;
   padding: 0;
+  overflow-x: auto;
+  overflow-x: overlay;
+  scroll-snap-type: x mandatory;
```

Comment:
So does it do anything? I barely could notice a difference with and without it,,

Reply:
a ok... mobile devices..
Need to read description twice..